packages/libs/react-ui/src/components/Notification/Notification.css.ts

```diff
@@ -51,7 +51,7 @@ export const cardColorVariants = styleVariants(colorVariants, (color) => {
 
 export const expandVariants = styleVariants({
   true: [sprinkles({ width: '100%', maxWidth: '100%' })],
-  false: [sprinkles({ width: 'max-content', maxWidth: 'maxContent' })],
+  false: [sprinkles({ width: 'max-content', maxWidth: '$maxContent' })],
```

Comment:
is the width not `$maxContent`? like maxWidth?

Reply:
No, because at least this version of the notifications component isn't supposed to be always be a fixed size. I believe this will likely change in the future though